src/routes/balances/converters.rs

```diff
@@ -43,8 +43,8 @@ impl BalanceDto {
                 logo_uri,
             },
             balance: self.balance.to_owned(),
-            fiat_balance: fiat_balance.to_string(),
-            fiat_conversion: fiat_conversion.to_string(),
+            fiat_balance: ((fiat_balance * 10000.0).round() / 10000.0).to_string(),
+            fiat_conversion: ((fiat_conversion * 10000.0).round() / 10000.0).to_string(),
```

Comment:
This solution should work but has one issue regarding the value that its representing: for fiat balances I would try to not use `round()` as it can round up and thus it'll show a higher value then it actually is.

In this case it's better to use `floor()`.

I also suggest extracting the rounding scale to a private variable in the file like

```rust
const SCALE: f64 = 1_0000_f64;

impl BalanceDto {
  ...
```

So that we know upfront which scale we are using.

Reply:
Ok agree. There is another way to set the number of decimals to 4 and avoid the division, that is using format. 
The issue for me is that if for example we have to round 1.1, format is returning 1.1000.
This result was breaking a test that expect a result with two decimals. What do you think is preferable use the division or we can accept the trailing zeroes and avoid the division?

Reply:
> Ok agree. There is another way to set the number of decimals to 4 and avoid the division, that is using format.
> The issue for me is that if for example we have to round 1.1, format is returning 1.1000.

Have you tried using `format!("{:.4}", number_to_be_formatted)`? This should format the number to use 4 decimal places but I think it still causes rounding (up or down) and we want to use `floor` (if you find a way around this let me know 😄 )

> This result was breaking a test that expect a result with two decimals.

But the unit tests were successful for this change 🤔 Do you remember which test broke? It should have been executed.

> What do you think is preferable use the division or we can accept the trailing zeroes and avoid the division?

Since the end result should be a `String` my preference goes to `format!()` but only if it "executes" `floor` and not `round`.

The division is fine 👍

Reply:
> But the unit tests were successful for this change thinking Do you remember which test broke? It should have been executed.
``` 
---- common::converters::tests::balances::native_token_balance stdout ----
thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `Balance { token_info: TokenInfo { token_type: NativeToken, address: "0x0000000000000000000000000000000000000000", decimals: 18, symbol: "ETH", name: "Ether", logo_uri: Some("https://test.token.image.url") }, balance: "7457594371050000001", fiat_balance: "2523.7991", fiat_conversion: "338.4200" }`,
 right: `Balance { token_info: TokenInfo { token_type: NativeToken, address: "0x0000000000000000000000000000000000000000", decimals: 18, symbol: "ETH", name: "Ether", logo_uri: Some("https://test.token.image.url") }, balance: "7457594371050000001", fiat_balance: "2523.7991", fiat_conversion: "338.42" }`', src/common/converters/tests/balances.rs:35:5
``` 
Fails because format! is adding zeroes until fill the number of decimals **fiat_conversion: "338.4200" !=  fiat_conversion: "338.42"**